.github/workflows/package.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request: {branches: [main]}
   schedule:
     - cron: '0 0 * * MON'
+  workflow_dispatch:
```

Comment:
The cron schedule lapsed due to no activity for 6 months, and I could not manually trigger a test run without a workflow dispatch hook.